module/user-extension/helper.js

```diff
@@ -681,8 +681,53 @@ module.exports = class UserExtensionHelper {
             }
         })
     }
+    
+    static findOne(query){
```

Comment:
we dont have any function ? @praveenKDass

Reply:
resolved